tests/std/tests/P0896R4_views_iota/test.cpp

```diff
@@ -387,6 +406,25 @@ int main() {
         auto r = views::iota(ranges::begin(f), ranges::end(f));
 
         assert(ranges::equal(r, even_ints, ranges::equal_to{}, deref));
+
+        using FilteredIota = decltype(r);
+        static_assert(CanEmpty<FilteredIota>);
+        static_assert(CanMemberEmpty<FilteredIota>);
+    }
+    // LWG-4001 iota_view should provide empty
+    {
+        using BackInsertingIota = decltype(views::iota(back_inserter(declval<vector<int>&>())));

```

Comment:
Should include `<iterator>` for `back_inserter()`. (\[iterator.range\] guarantees that `<vector>` drags in non-member `begin()` etc. but not the rest of `<iterator>`.)

Reply:
[`<ranges>` includes `<iterator>`](https://eel.is/c++draft/ranges.syn).